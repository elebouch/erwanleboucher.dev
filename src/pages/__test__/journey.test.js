import React from "react"
import { render } from "@testing-library/react"
import "jest-styled-components"

import Journey from "../journey"

describe("Journey", () => {
  it("renders correctly", () => {
    const { container } = render(<Journey />)
    expect(container.innerHTML).toMatchSnapshot()
  })
})
