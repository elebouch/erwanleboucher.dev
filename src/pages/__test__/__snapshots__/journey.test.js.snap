// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Journey renders correctly 1`] = `"<article><div class=\\"sc-AxhUy hSpNqt\\"><div class=\\"sc-AxhCb cXBSVz\\"></div><span class=\\"sc-AxgMl eEdZGe\\">Work experience</span><div class=\\"sc-AxjAm cFQoNO\\"><p class=\\"sc-AxirZ eSQAuU\\">viafintech | Software Engineer</p><p class=\\"sc-AxiKw icjPNi\\">September 2019 – Present | Berlin Germany</p><span>viafintech is a payment service provider. Through viacash, they offer a solution for cash deposits, withdrawals and payment for clients like N26, bunq, Amazon.de in retails such as DM, Rewe, Penny.</span><ul><li>Integrated Alipay payment solution into their payment platform.</li><li>Designed and developed an iOS and Android application to find the nearest retail partners with React Native.</li></ul>(Golang, Ruby on Rails, Postgresql, RabbitMQ)</div><div class=\\"sc-AxjAm cFQoNO\\"><p class=\\"sc-AxirZ eSQAuU\\">GitGuardian | Full-Stack Developer</p><p class=\\"sc-AxiKw icjPNi\\">November 2018 – July 2019 | Paris, France</p><span>GitGuardian is a solution that scans GitHub activity in real-time for API secret tokens, database credentials, certificates and alerts the developer within a minute.</span><ul><li>Involved in the main website redesign, now viewed by more than 50k people per month.</li><li>Built a B2B dashboard used by some Fortune 500 with Django and React.</li></ul>(Django, Python, React, Postgresql, Golang)</div></div><div class=\\"sc-AxhUy hSpNqt\\"><div class=\\"sc-AxhCb cXBSVz\\"></div><span class=\\"sc-AxgMl eEdZGe\\">Involvements &amp; Hackathons</span><div class=\\"sc-AxjAm cFQoNO\\"><p class=\\"sc-AxirZ eSQAuU\\">Android OS open-source community</p><p class=\\"sc-AxiKw icjPNi\\">February 2015 – November 2017</p>Adding improvements and features on the Android system and kernel, with Java and C. Member of multiple organizations such as LineageOS/Cyanogenmod and AICP. Recognized developer on the developer forum XDA.</div><div class=\\"sc-AxjAm cFQoNO\\"><p class=\\"sc-AxirZ eSQAuU\\">Navya x 42 - Team Exedra - First Place</p><p class=\\"sc-AxiKw icjPNi\\">Mondial Paris Motor Show 2018 | CES 2019</p>Navya organized a contest with 42 about improving the user experience aboard the car. We envisioned a pop-up meeting room, then we engineered a platform to book and manage meetings with React Native and Express.</div></div><div class=\\"sc-AxhUy hSpNqt\\"><div class=\\"sc-AxhCb cXBSVz\\"></div><span class=\\"sc-AxgMl eEdZGe\\">Education</span><div class=\\"sc-AxjAm cFQoNO\\"><p class=\\"sc-AxirZ eSQAuU\\">Ecole 42</p><p class=\\"sc-AxiKw icjPNi\\">2019</p></div></div></article>"`;
